pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -53,6 +53,16 @@ const (
 	// DefaultMaxFileSize is the default maximum file size configuration value that will be used unless nginx annotation is set
 	DefaultMaxFileSize = 1000
 
+	// MattermostAppContainerName is the name of the container which runs Mattermost application
+	MattermostAppContainerName = "mattermost"
+
+	// CertificateFilesSecretName is the name of a secret used for mounting CA certificate files in the
+	// the container that runs the Mattermos application.
+	CertificateFilesSecretName = "ca-certificate-files"
+	// CertificateFilesPath is the path where the certificate files will be mounted. Note that this path
+	// is valid for Debian and Ubuntu but may not work for other distributions.
```

Comment:
we use Alpine as base image.

Reply:
It should be the same for Alpine. Are the container images used in dev also built from this distribution?

Reply:
everything and alpine have the same path, just checked